feat(docs): `as bytes32` and `as bytes64` storage annotations (#2748)
tact-lang/tact: docs/src/content/docs/book/cells.mdx

:::

### `bytes32` {#serialization-bytes32}

> The `as bytes32` serialization annotation can only be applied to slices and is used to read 32 bytes (256 bits) of data from a cell. The `as uint256` annotation achieves a similar result, but `as uint256` can only be applied to integers.

Reviewer: ```suggestion
The `as bytes32{:tact}` serialization annotation can only be applied to slices and is used to read 32 bytes (256 bits) of data from a cell. The `as uint256{:tact}` annotation achieves a similar result, but it can only be applied to integers.
```

> The `as bytes32` serialization annotation can only be applied to slices and is used to read 32 bytes (256 bits) of data from a cell. The `as uint256` annotation achieves a similar result, but the latter can only be applied to integers, while the former can be applied to a slice.

Reviewer 2: let's not use latter/former:

The `as uint256` annotation achieves a similar result, but the `as uint256` can only be applied to integers, while the `as bytes32` can only be applied to a slice.

Reviewer: The `as uint256` annotation achieves a similar result, but the it can only be applied to integers, while `as bytes32` can only be applied to a slice.

> The `as bytes32` serialization annotation can only be applied to slices and is used to read 32 bytes (256 bits) of data from a cell. The `as uint256` annotation achieves a similar result, but the latter can only be applied to integers, while the former can be applied to a slice.

Author: ```suggestion
The `as bytes32` serialization annotation can only be applied to slices and is used to read 32 bytes (256 bits) of data from a cell. The `as uint256` annotation achieves a similar result, but `as uint256` can only be applied to integers.
```

### `bytes64` {#serialization-bytes64}

> The `as bytes64` serialization annotation can only be applied to slices and is used to read 64 bytes (512 bits) of data from a cell. It can be useful for serializing and deserializing cryptographic signatures. For a usage example, see the [`SignedBundle`](/ref/core-crypto#signedbundle) idiom.

Reviewer: ```suggestion
The `as bytes64{:tact}` serialization annotation can only be applied to slices and is used to read 64 bytes (512 bits) of data from a cell. It can be useful for serializing and deserializing cryptographic signatures.

For a usage example, see the [`SignedBundle`](/ref/core-crypto#signedbundle) [struct][struct].
```
